test/ossl_shim/packeted_bio.cc

```diff
@@ -240,7 +240,7 @@ static int PacketedFree(BIO *bio) {
   return 1;
 }
 
-static long PacketedCallbackCtrl(BIO *bio, int cmd, bio_info_cb fp) {
+static long PacketedCallbackCtrl(BIO *bio, int cmd, BIO_info_cb fp) {
```

Comment:
same curly comment.

Reply:
okay.

Reply:
BTW I wonder if this C++ gets compiled at all?

Reply:
yes, if you run the extended boringssl tests.  (I had the same question last week)